frontends/mit-open/src/pages/DashboardPage/Dashboard.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { renderTestApp, screen, waitFor } from "../../test-utils"
+
+describe("DashboardPage", () => {
+  test("Renders title", async () => {
+    renderTestApp({
+      url: `/dashboard`,
```

Comment:
Do we favor double quotes over template delimiters where there is no interpolation?

We can enforce with ESLint - `quotes: ["error", "double", { avoidEscape: true }]` (`avoidEscape` is needed if we want e.g. `'abc"def'` without fighting with Prettier). The base config [defers to prettier](https://github.com/mitodl/eslint-config/blob/master/index.js#L34C5-L34C68), though it doesn't seem Prettier can fix to double quote preference over backticks.